libexec/drivers/coffeetags

```diff
@@ -27,9 +27,13 @@
 #
 #
 
-if ! type coffeetags > /dev/null 2>&1; then
-    exit 77
-fi
+{
+    if  [[ (! $(type coffeetags)) || \
+        ($(type rbenv) && ! $(rbenv which coffeetags)) ]]
```

Comment:
`[[ ]]` is a bashism, and this script uses `/bin/sh` so shouldn't use bashisms (or should be a bash script).  Also, I'm not so familiar with `[[ ]]` but it looks a bit weird here… does this check the output of the commands and all the test are for (non) emptiness of the output??

Some testing confirms that a POSIX variant could be:
```shell
[ -z "$(type coffeetags)" -o -n "$(type rbenv)" -a -z "$(rbenv which coffeetags)" ]
```

However, couldn't we just use a normal shell construct, like this?  Only thing it needs is for the various commands to return a meaningful value.
```shell
type rbenv && ! rbenv which coffeetags || ! type coffeetags
```

Anyway, the check looks a bit off to me, because I would imagine someone could be running outside of rbenv yet having rbenv in the PATH, couldn't it?  If so, we'd have to check we are *actually* in a rbenv… anyway, I don't know anything about Ruby, so I might be wrong -- and I don't mind much.

Also, I don't think the test is even exactly the one you intended.  IIUC, what you want is "if `type rbenv` succeeds, then exit 77 if `rbenv which coffeetags` fails.  Else, if `type rbenv` fails, return 77 if `type coffeetags` fails".  This is not exactly what you have here, while it'll probably work in practice if as you say `type coffeescript` always succeed if `rbenv which coffeetags` does. What you have is "if `type coffeetags` fails or if `type rbenv` succeeds and `rbenv which coffeetags` fails, return 77", which means that if both `type rbenv` and `rbenv which coffeetags` succeed but `type coffeetags` does not, return 77.